templates/ci/python-django.yml

```diff
@@ -0,0 +1,47 @@
+# Python Django
+# Test a Django app on multiple versions of Python.
+# https://aka.ms/yaml
+
+phases:
+
+# =================
+# Test a Django app
+# =================
+- phase: 'DjangoApp'
+  variables:
+    # relative path to your manage.py
+    projectPath: mysite/
```

Comment:
Perhaps move the comment to the right of  `projectPath: mysite/` to make it clear that it doesn't apply to additional variables that might get added.  Also, what do you think about removing the word 'your' so that it doesn't look third-partyish if people adopt it long-term?

Reply:
:+1: